src/editors/containers/TextEditor/hooks.js

```diff
@@ -173,11 +173,44 @@ export const prepareEditorRef = () => {
   return { editorRef, refReady, setEditorRef };
 };
 
-export const getContent = ({ editorRef, isRaw }) => () => {
-  if (isRaw && editorRef && editorRef.current) {
-    return editorRef.current.state.doc.toString();
-  }
-  return editorRef.current?.getContent();
+export const setAssetToStaticUrl = ({ editorValue, images }) => {
```

Comment:
Did you test to see if this still works in the text editor? Just don't want to regress behavior

Reply:
the behavior is as expected for the text editor